test_cli.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# Copyright 2021 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+
+CMD=./chainbridge-core-example
+
+KALEIDO_NODE_URL="wss://u0oo16l2hl:WQgUMKVenv9mefKXW57fV-TzSWhFz06YILiXlHGykd4@u0ize6fsgo-u0np93mhu4-wss.us0-aws.kaleido.io"
```

Comment:
Could we have these variables be passed into the script? Or are we planning to have people just edit the file directly before running it?

Reply:
Same answer as above. Going to split PRs and yeah, URL can be env variable or passed in as argument instead of hardcoded for extensibility, though at the same time perhaps we just leave values blank for final public version since contract addresses will also need to be replaced and that would require either too many env vars or command line arguments

Reply:
I'll wipe our test values, generalize `NODE_URL` and users can customize it